node/resolvers/Routes/index.ts

```diff
@@ -38,6 +38,7 @@ const getUserAndPermissions = async (ctx: Context) => {
   const {
     data: { checkUserPermission },
   }: any = await storefrontPermissions
+    // I don't know why this name sender is necessary, maybe it is some policy or permission.
```

Comment:
nit: Can we update this comment with the information on the PR discussion? might be useful in the future.